package.json

```diff
@@ -5,9 +5,9 @@
   "main": "dist/index.js",
   "scripts": {
     "prestart": "npm install",
-    "start": "webpack-dev-server --progress",
+    "start": "node_modules/.bin/webpack-dev-server --progress",
```

Comment:
What I meant is that we install it locally and you don't need to give the full path, it works with `webpack-dev-server` as is. And if you do have it installed globally, Webpack looks for the local installed module any way.